.ci/deps.sh

```diff
@@ -35,6 +35,7 @@ mkdir -p ~/.RLibrary
 echo '.libPaths( c( "~/.RLibrary", .libPaths()) )' >> .Rprofile
 echo 'options(repos=structure(c(CRAN="http://cran.rstudio.com")))' >> .Rprofile
 R -e "install.packages('lintr', dependencies=TRUE, quiet=TRUE, verbose=FALSE)"
+R -e "install.packages('formatR', dependencies=TRUE, quiet=TRUE, verbose=FALSE)"
```

Comment:
and I would also remove inside the commit message "... by FormatRBear **R-file**", this "R-file" doesn't really help imo :+1: